recipes/enstat/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "enstat" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f5c7a3dbb76ae4364ae4271212851b6655fb868481c9715163c781c8eb5d44cb
+
+build:
+  number: 0
+  noarch: python
+  script_env: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3
+    - numpy
+
+test:
+  imports:
+    - enstat
```

Is there any reason for not including `pip check` here?
```suggestion
    - enstat
  commands:
    - pip check
  requires:
    - pip
```